app/models/__init__.py

```diff
@@ -1 +1,2 @@
-from .table import Tables
\ No newline at end of file
+from .table import Tables
+from.column import Columns
```

Comment:
i think there is a space issue. Check again.

Reply:
done